packages/create-vite/template-react-ts/tsconfig.json

```
@@ -16,5 +16,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": ["./src"]
+  "include": ["./src", "vite.config.ts"]
```

I wonder if we should drop `./` for consistency with the other templates:

```suggestion
  "include": ["src", "vite.config.ts"]
```

Seems like it got added in https://github.com/vitejs/vite/commit/245303ca35ff2a40eca49e102b4f82cb1210f597#diff-0b5311c6ba7c21254b69cb89944f526d3481da8db60632f466f1f69ad979b11d but I can't think of a good reason for it. Next.js also generates a `tsconfig.json` without `./`